src/main/java/study/racing/step3/Racing.java

```diff
@@ -0,0 +1,25 @@
+package study.racing.step3;
+
+import java.util.List;
+
+public class Racing {
+
+    public static void main(String[] args) {
+        int carCount = View.inputView.input("자동차 대수는 몇 대 인가요?");
+        int loopCount = View.inputView.input("시도할 회수는 몇 회 인가요?");
+
+        List<Car> cars = CarFactory.createCars(carCount, new RandomMovingCondition());
+
+        for (int i = 0; i < loopCount; i++) {
+            cars.stream().forEach(car -> {
```

Comment:
```suggestion
            cars.forEach(car -> {
```
forEach를 사용하는 경우라면 stream을 사용하실 필요없이 그냥 사용가능합니다 ㅎㅎ